### ISSUE TITLE
[CS] [HL25] Wrong weapon sway when moving

### ISSUE THREAD

**Issue by h-a-s-k** [ValveSoftware/halflife#3691]
### Game Version
Protocol version 48
Exe version 1.1.2.7/Stdio (cstrike)
Exe build: 01:03:30 Dec  1 2023 (9899)

### Description

When moving after the anniversary update the animation for weapon sway/bob/rollangle/whatever you wanna call it is wrong.
If I had to guess it's using the half-life one instead of the proper 1.6 animation, notice how it moves to the side instead of back and forth as it should be.

Affects every weapon I've tested and `sv_rollangle 0` does not fix it.

### Comparison

https://github.com/ValveSoftware/halflife/assets/140975143/7ae86e3a-75f9-4e0a-bf77-2836dd5bafd1

https://github.com/ValveSoftware/halflife/assets/140975143/759bb567-5c2d-4a62-9e6c-920c3f8ec799

**Comment by di57inct** [comment]
when is this going to be fixed? we need original bob movement for cs or at least add a setting for it.

**Comment by JXGSx101** [comment]
Is this going to be fixed? 
Edit: This also applies to CS Condition Zero.

**Comment by di57inct** [comment]
sorry for the ping @kisak-valve but it has been over a year and it just isn't 1.6 anymore with that bobbing style. can we please have our old bob back?

**Comment by sabianroberts** [comment]
This is how Counter-Strike behaved when it first released. It has been restored to its original behaviour. If it bothers you then do `cl_bob 0` in the console

**Comment by di57inct** [comment]
> This is how Counter-Strike behaved when it first released. It has been restored to its original behaviour. If it bothers you then do `cl_bob 0` in the console

it's also how it didn't behave for the past 20+ years. we should at least be able to choose the style we want.

**Comment by JXGSx101** [comment]
> This is how Counter-Strike behaved when it first released. It has been restored to its original behaviour. If it bothers you then do `cl_bob 0` in the console

Wrong. When CS officially released in 2000 it already had the original/steam_legacy viewbob. CS only had the Half Life Retail Viewbob during the beta 1.0 to 6.1. After the update 1.1.1.0 from Half Life, the retail viewbob was changed to the original/steam_legacy viewbob that has been used for over 25 years.

**Comment by JXGSx101** [comment]
It's been three years and it still hasn't been fixed.
By the way, if you playback a demo on the 25th anniversary version the viewbob is the original one.
I reeinforce that this is a problem in Counter-Strike: Condition Zero.
@kisak-valve 

https://github.com/user-attachments/assets/8a74d768-1b1a-409c-be1f-6c703135f6bf
